### PR TITLE
Modify copyright comment

### PR DESCRIPTION
**Pull request by PengYiOfOpenSDS**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Solve Issue [#273](https://github.com/opensds/opensds/issues/273)
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by leonwanghui** [comment]
LGTM

File: osdsctl/completion/osdsctl.bash_completion [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+
+# Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```
Comment (xing-yang) [inline osdsctl/completion/osdsctl.bash_completion]: Apache license?

File: pkg/dock/proto/dock.pb.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```
Comment (xing-yang) [inline pkg/dock/proto/dock.pb.go]: Apache?

File: osdsctl/cli/version_test.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The OpenSDS Authors.
+// Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (PengYiOfOpenSDS) [reply osdsctl/cli/version_test.go]: Because this file was added in 2018.

**Comment by leonwanghui** [comment]
Please wait a moment for others to review

**Pull request merged**